doc/nrf/doc_build.rst

```diff
@@ -148,25 +148,25 @@ To clean the build folders for the Zephyr documentation:
 
 .. code-block:: console
 
-   ninja clean-zephyr
+   ninja zephyr-clean
```

Comment:
How important is it to be able to clean each individual step ?
Wouldn't it be safer to use the already existing target `ninja clean` and then do a complete clean instead.

Reply:
It's required - we don't want to build for 20 minutes every time we need a clean nrf or nrfxlib doc build.

Reply:
>  Wouldn't it be safer to use the already existing target `ninja clean` and then do a complete clean instead.

Safer, but it is so very, very slow. I think it's nice to be able to clean individual pieces and not lose the other docsets.

Reply:
I have setup `ninja clean` so that it cleans all artifacts, but individual cleans are still available.